bin/setup

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+# This assumes you have general prerequisites installed as by:
+# https://github.com/artsy/potential/blob/master/scripts/setup
+
+# Exit if any subcommand fails
+set -e
+
+echo "brew update and bundle install..."
+brew update
+brew bundle --file=- <<EOF
+brew asdf
+brew node
```

Comment:
Not sure about this. Maybe just use system's `node` (select via `nvm` which is installed by Potential setup script)?